builtin/game/falling.lua

```diff
@@ -170,6 +175,9 @@ local function drop_attached_node(p)
 		drops = drop_stacks
 		def.preserve_metadata(pos_copy, node_copy, oldmeta, drops)
 	end
+	if def and def.sounds and def.sounds.fall and def.sounds.fall.name then
```

Comment:
Must not be a table. `core.sound_play("sound_name", { pos = p })` for example, is valid too.

Reply:
You wanted to write "Does not need to be a table.", didn't you?
Falling node object becoming a node does the same:
https://github.com/minetest/minetest/blob/766fb7b46edab47f96b8b940e390daf5319261b7/builtin/game/falling.lua#L119-L121

Reply:
Yes, and yes please.